client-node-tests/src/servers/testServer.ts

```diff
@@ -527,9 +527,10 @@ connection.onRequest(
 	async (_, __) => {
 		const progressToken = 'TEST-PROGRESS-TOKEN';
 		await connection.sendRequest(WorkDoneProgressCreateRequest.type, { token: progressToken });
-		void connection.sendProgress(WorkDoneProgress.type, progressToken, { kind: 'begin', title: 'Test Progress' });
```

Comment:
Could we please keep both tests and clarify for the test with fractions that this is outside of the spec but that we have rounding code to tolerate things since JS has not real integer support.